docs/developer-guide.rst

```diff
@@ -72,48 +84,50 @@ Cobbler has a development branch called "master" (where the action is), and bran
 
-* Always use under_scores, not camelCase.
-* Always four (4) spaces, not tabs.
-* Avoid one line if statements.
-* Validate your code by using ``make qa``.
-* Keep things simple, keep in mind that this is a tool for sysadmins and not python developers.
-* Use modules that are easily available (eg. EPEL) but preferrably in the base OS, otherwise they have to be packaged with the app, which usually runs afoul of distribution packaging guidelines.
-* At least for now we have to support Python 2.7 for Cobbler and ython 2.6 for Koan.
-
-You're also welcome to hang out in #cobbler and #cobbler-devel on irc.freenode.net, as there are folks around to answer questions, etc.
+- Always use under_scores, not camelCase.
+- Always four (4) spaces, not tabs.
+- Avoid one line if statements.
+- Validate your code by using ``make qa``.
+- Keep things simple, keep in mind that this is a tool for sysadmins and not python developers.
+- Use modules that are easily available (eg. EPEL) but preferrably in the base OS, otherwise they have to be packaged
+  with the app, which usually runs afoul of distribution packaging guidelines.
+- At least for now we have to support Python 2.7 for Cobbler and Python 2.6 for Koan.
 
+You're also welcome to hang out in #cobbler and #cobbler-devel on irc.freenode.net, as there are folks around to answer
+questions, etc. But it isn't that active anymore please drop also in our Cobbler Gitter channel there we will probably
+answer faster.
 
 Contributing to the website
-+++++++++++++++++++++++++++
+###########################
 
-The github-based git repository for the https://cobbler.github.io website itself is at https://github.com/cobbler/cobbler.github.io.
+The github-based git repository for the https://cobbler.github.io website itself is at
+https://github.com/cobbler/cobbler.github.io.
 
 If you want to contribute changes to the website, you will need jekyll (http://jekyllrb.com).
 
 You will probably want to:
 
-* edit the files in _dynamic
-* run the generate_dynamic.sh script
-* add both the .md and resulting .html files in your git commit
-
+- edit the files as markdown
+- run the docker container
+- check if your changes didn't break anything
 
 Mailing List
-++++++++++++
+############
 
 We have a development mailing list at https://fedorahosted.org/mailman/listinfo/cobbler-devel
 Discuss development related questions, roadmap, and other things there, rather than on the general user list.
 
 It is a very good idea to mention your pull request (copy/paste, etc) to the development mailing list for discussion.
 
-
 Debugging
-+++++++++
+#########
 
-If you need to debug a remote process, epdb provides some very nice capabilities beyond the standard python debugger, just insert a "import epdb; epdb.serve()" in your command line, and from the console:
+If you need to debug a remote process, epdb provides some very nice capabilities beyond the standard python debugger,
```

Comment:
`epdb` is cool, but some folks might find it easier to navigate with `rpdb`, since it's close to `pub`. Maybe we should also mention this.

Reply:
Fixed with 7fc4a13